smart_kit/models/smartapp_model.py

```diff
@@ -52,6 +53,11 @@ def __init__(self, resources: SmartAppResources, dialogue_manager_cls, custom_se
     def get_handler(self, message_type):
         return self._handlers[message_type]
 
+    def init_additional_handlers(self):
+        self._handlers.update({
+            message_name: handler(self.app_name) for message_name, handler in self.additional_handlers.items()
```

Comment:
К комментарию Артёма про дополнительные аргументы:
```suggestion
            message_name: handler_params["handler"](self.app_name, **handler_params.get("params", {})) for message_name, handler_params in self.additional_handlers.items()
```
где формат `additional_handlers` примерно такой:
{"MESSAGE_TO_SKILL": {"handler": HandlerText, "params": {"dialogue_manager": self.dialogue_manager}}}

Reply:
Го. @Dan1lD норм?